keyboards/basekeys/trifecta/config.h

```diff
@@ -0,0 +1,127 @@
+/* Copyright 2020 Swiftrax and Basekeys.com
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x04D8
+#define PRODUCT_ID      0xEAF3
```

Comment:
Could you use a different VID here?  This belongs to Microchip. 
https://www.the-sz.com/products/usbid/index.php?v=0x04D8&p=&n=

For instance: 
```suggestion
#define VENDOR_ID       0x6541
#define PRODUCT_ID      0x4277
```

Reply:
I received this valid vid and pid from Microchips USB sublicensing program. You can view the application process here:
https://www.microchip.com/usblicensing